utils/convert.mjs

```diff
@@ -536,7 +529,14 @@ function cleanUpClassItems(data) {
         type = `...${type}[]`;
       }
       return type;
-    }
+    };
+
+    const processOverload = overload => {
+      if (overload.params) {
+        return Object.values(overload.params).map(param => processParam(param));
+      }
+      return overload;
+    };
```

Comment:
ESLint warned that `processOverload` comes before `processParam`, thus it might call uninitialized variable. This is why I moved this code.